app/src/main/kotlin/com/saltedge/authenticator/app/Constants.kt

```diff
@@ -36,13 +36,12 @@ const val KEY_GUID = "guid"
 const val KEY_OPTION_ID = "option_id"
 const val KEY_ACTION = "action"
 const val KEY_DEEP_LINK = "KEY_DEEP_LINK"
-const val KEY_CONNECTION_GUID = "connection_guid"
+const val KEY_STATUS_CLOSED = "closed"
```

Comment:
I think that you don't need to define this key. You already have `AuthorizationStatus` enum. You should add `closed` status there.

Reply:
Since we don't perform any actions with this status, it will be redundant to add it for now.